pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -197,6 +197,17 @@ func (r *ReconcileComplianceScan) phaseLaunchingHandler(instance *compv1alpha1.C
 	}
 
 	if err = r.createScanPods(instance, nodes, logger); err != nil {
+		if !common.IsRetriable(err) {
```

Comment:
I like surfacing the error, but wouldn't it be better to do it in the main `Reconcile` loop entry point?

Reply:
Mind if I do that in a separate PR? That needs further refactoring as we need to handle when the result has a requeue.

Reply:
sure